microfrontends/app-builder-menu/src/components/Menu.tsx

```diff
@@ -1,33 +1,94 @@
-import axios from 'axios';
-import styled from 'styled-components';
 import { useEffect, useState } from 'react';
+import axios from 'axios';
+import MenuUI, { MfeConfig } from './MenuUI';
+import { createGlobalStyle } from 'styled-components';
+import { MENU_ITEM, PBC_API_RESPONSE } from '../types/api';
+import { BrowserRouter } from 'react-router-dom';
 
-const MenuCmp = styled.menu`
-  position: fixed;
-  left: 0;
-  top: 0;
-  background-color: black;
-  width: 250px;
-  height: 100%;
-  color: white;
+const GlobalStyle = createGlobalStyle`
```

Comment:
what if we move it to a `styles.ts` file?

Reply:
GlobalStyled will be moved in the end, so does not make sense to move it now.